docs/source/guide/configuration.rst

```diff
@@ -466,6 +471,10 @@ in the ``~/.aws/config`` file.
           functionality of ``standard`` mode with automatic client-side
           throttling.  This is a provisional mode whose behavior might change.
 
+``sigv4a_signing_region_set``
+A comma delimited list of regions use when signing with SigV4a. If this is not set,
+the SDK will check if the service has modeled a default; if none is found, this will
```

Comment:
suggestion: 'If this is not set, the SDK will use a default value modeled by the service, if any; if no default value is provided by the service, this will default to ``*``, which indicates the request is valid in all regions.'